stdlib/messaging/rabbitmq/src/main/java/org/ballerinalang/messaging/rabbitmq/util/ConnectionUtils.java

```diff
@@ -178,17 +172,16 @@ private static SSLContext getSSLContext(MapValue secureSocket) {
         }
     }
 
-    /**
-     * Handles closing the given connection.
-     *
-     * @param connection   RabbitMQ Connection object.
-     * @param timeout      Timeout (in milliseconds) for completing all the close-related
-     *                     operations, use -1 for infinity.
-     * @param closeCode    The close code (See under "Reply Codes" in the AMQP specification).
-     * @param closeMessage A message indicating the reason for closing the connection.
-     */
-    public static void handleCloseConnection(Connection connection, Object closeCode, Object closeMessage,
-                                             Object timeout) {
+    public static boolean nativeIsClosed(Connection connection) {
+        boolean flag = false;
+        if (connection == null || !connection.isOpen()) {
+            flag = true;
+        }
+        return flag;
```

Comment:
can write this in a single line:
```suggestion
        return connection == null || !connection.isOpen();
```

Reply:
Fixed.